### DISCUSSION TITLE
Wrong house consumption Master + Slave

### DISCUSSION THREAD

**Discussion by Tobim99** [weltenwort/home-assistant-rct-power-integration#375] [Q&A]
Hey everyone. I have a question about house consumption. In the readme it is documented that the entity "Consumer Power" shows the power currently consumed by the household. Unfortunately, this is not the case for me. The RCT-Power app shows me a house consumption of 585W and in Home Assistant -1,215W.

I have a master (PS 10.0 IL89) and a slave (PS 6.0 6MDQ) with software 2.3.5661.

Does anyone have any idea how I can get the current house consumption? 

![RCT-Powe App](https://github.com/weltenwort/home-assistant-rct-power-integration/assets/23269864/b9f7295f-4110-4af8-adef-93c37807a89c)
![HA master consumer power](https://github.com/weltenwort/home-assistant-rct-power-integration/assets/23269864/8913dbf5-8bf7-4743-bd35-c1e6c226ddcb)

**Comment by weltenwort** [comment]
Hi @Tobim99, thanks for reporting this. Unfortunately there's no official documentation on the meaning of the various registers, so this integration might misinterpret some of them. Would you be able to share a screenshot of the detail pages from the App showing the household and external generator pages as well as a snapshot of the power-related integration entities? Maybe we can figure out how they correlate this way.

**Comment by benzoesaeure** [comment]
Hi @Tobim99 , ich schreibe hier mal auf deutsch da ich nicht genau weiß wich das richtig in Englisch ausdrücken soll. ich habe das gleiche Problem und auch letztes Jahr eine Anfrage geschrieben. Das Problem sind die zwei Wechselrichter. Ich habe mittlerweile meine beden DC 6 in der HA Integration hinzugefügt. Weiter habe ich einen Helfer in HA erstellt  (Kombiniere den Zustand mehrerer Sensoren) und habe da aus beiden Wechselrichtern die Sensoren sensor.rct_power_storage_consumer_power und sensor.rct_power_storage_consumer_power_2 eingetragen und auf summieren gestellt. So kommst du auf den aktuellen verbrauch.
